### DISCUSSION TITLE
note/node-api-method-summary/

### DISCUSSION THREAD

**Discussion by giscus** [gitguanqi/gitguanqi.github.io#13] [Announcements]
# note/node-api-method-summary/

导语：之前做了一个小项目，其中后端接口中封装了很多的方法，现在就总结罗列一下，方便保存，以便日后用到。
目录
数据库
Node相关
数据处理
文件处理
数据库
mysql连接
`js
// mysql.js
const mysql = require('mysql');
const pool = mysql.crea ...

https://guanqi.xyz/note/node-api-method-summary/

**Comment by gitguanqi** [comment]
写的挺好的
